src/Compilers/VisualBasic/Portable/Language/LanguageFeature.vb

```diff
@@ -173,9 +185,21 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
         Private Function IsInLanguageVersion(feature As Feature, options As VisualBasicParseOptions) As Boolean
             Dim required = feature.GetLanguageVersion()
             Dim current = options.LanguageVersion
-            Return required <= current
+            Return CheckVersionNumbers(required, current)
         End Function
 
+        <Extension>
+        Friend Function IsInLanguageVersion(feature As Feature, options As VisualBasicCompilation) As Boolean
+            Dim required = feature.GetLanguageVersion()
+            Dim current = options.LanguageVersion
+            Return CheckVersionNumbers(required, current)
+        End Function
+
+        Private Function CheckVersionNumbers(required As LanguageVersion, current As LanguageVersion) As Boolean
+            Return (Integer.MinValue < required) AndAlso (required <= current)
```

Comment:
`Integer,MinValue` is used as guard against features still being developed. eg `IOperation`